applications/solvcon/conda.sh

```diff
@@ -1,8 +1,8 @@
 #!/bin/bash
 conda install -y \
```

Comment:
I wonder why we still need conda?

Reply:
`conda` builds packages that are not built by `devenv` yet. The current solution is a mixed solution of `conda` and `devenv`.

Reply:
Please list all packages that require conda in the checkin log.

Reply:
Sorry for responding late.  "Checkin log" is commit log.  Please add the list of packages that are needed to be installed by conda in one of the commit logs.